### PR TITLE
refresh data for us_eia_opendata

### PR DESCRIPTION
**Pull request by Harsha-chandaluri**

### CODE REVIEW

File: scripts/us_eia/opendata/process/common.py [hunk 1]
```diff
@@ -82,11 +234,13 @@ def _parse_date(d):
         m_or_q = d[4:]
 
         if m_or_q.startswith('Q'):
+            #print("withQ",yr + '-' + _QUARTER_MAP[m_or_q])
```
Comment (shamimansari1988) [inline scripts/us_eia/opendata/process/common.py:237]: Please remove the commented line if it's not required.

File: scripts/us_eia/opendata/process/common.py [hunk 2]
```diff
@@ -82,11 +234,13 @@ def _parse_date(d):
...
             # Quarterly
             if m_or_q in _QUARTER_MAP:
                 return yr + '-' + _QUARTER_MAP[m_or_q]
         else:
             # Monthly
+            #print("withOutQ",yr + '-' + m_or_q)
```
Comment (shamimansari1988) [inline scripts/us_eia/opendata/process/common.py:243]: Please remove the commented line if it's not required.

File: scripts/us_eia/opendata/process/common.py [hunk 3]
```diff
@@ -334,7 +512,9 @@ def process(dataset, dataset_name, in_json, out_csv, out_sv_mcf, out_svg_mcf,
 
             # Add to rows.
             rows = []
+            #print(time_series)
```
Comment (shamimansari1988) [inline scripts/us_eia/opendata/process/common.py:515]: Please remove the commented line from everywhere if it's not required.

File: scripts/us_eia/opendata/process.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    manifest_json = download_manifest()
+    datasets = manifest_json.get('dataset', {})
+    for dataset_name in datasets:
+        if FLAGS.dataset and dataset_name not in FLAGS.dataset:
+            continue
+        print(dataset_name)
+        dataset = datasets[dataset_name]
+        print("dataset", dataset)
+        download_file(dataset['accessURL'], f'{FLAGS.data_dir}/{dataset_name}')
+        print(f'{FLAGS.data_dir}/{dataset_name}')
```
Comment (shamimansari1988) [inline scripts/us_eia/opendata/process.py:79]: if print is really required then please change print to logging.info("............)  everywhere.

File: scripts/us_eia/opendata/process/README.md [hunk 5]
```diff
@@ -58,13 +58,13 @@ takes a raw stat-var and generates a fully defined stat-var for it.
...
 [manifest](https://api.eia.gov/bulk/manifest.txt) by running the
-[`download_bulk.py`](https://github.com/datacommonsorg/data/blob/master/scripts/us_eia/opendata/download_bulk.py)
+[`python3 process.py --dataset=TOTAL`](https://github.com/datacommonsorg/data/blob/master/scripts/us_eia/opendata/process.py)
 script.
 
 To generate CSV, TMCF and stat-var MCF for a supported dataset:
 
 ```bash
-python3 main.py --data_dir=tmp_raw_data/ELEC --dataset=ELEC
+python3 process.py --dataset=ELEC
```
Comment (shamimansari1988) [inline scripts/us_eia/opendata/process/README.md:67]: Please add the same for remaining imports with district parameters for each Data set.

File: scripts/us_eia/opendata/process/common.py [hunk 6]
```diff
@@ -21,7 +22,12 @@
 from sys import path
 
 # For import util.alpha2_to_dcid
-path.insert(1, '../../../../')
+# Setup path for import from data/util
+# or set export PYTHONPATH="./:<repo>/data/util" in bash
+#path.insert(1, '/usr/local/google/home/chandaluri/data/')
```
Comment (shamimansari1988) [inline scripts/us_eia/opendata/process/common.py:27]: Please remove if this line is not required.

Comment (Harsha-chandaluri) [reply scripts/us_eia/opendata/process/common.py:27]: removed

File: scripts/us_eia/opendata/process/README.md (hunk 5 above)
Comment (Harsha-chandaluri) [reply scripts/us_eia/opendata/process/README.md:67]: Added

File: scripts/us_eia/opendata/process.py (hunk 4 above)
Comment (Harsha-chandaluri) [reply scripts/us_eia/opendata/process.py:79]: Modified

File: scripts/us_eia/opendata/process/common.py (hunk 3 above)
Comment (Harsha-chandaluri) [reply scripts/us_eia/opendata/process/common.py:515]: Removed

File: scripts/us_eia/opendata/process/common.py (hunk 2 above)
Comment (Harsha-chandaluri) [reply scripts/us_eia/opendata/process/common.py:243]: Removed

File: scripts/us_eia/opendata/process/common.py (hunk 1 above)
Comment (Harsha-chandaluri) [reply scripts/us_eia/opendata/process/common.py:237]: Removed

File: scripts/us_eia/opendata/README.md [hunk 7]
```diff
@@ -22,9 +16,45 @@ This dataset is available for public use, license is available at https://www.ei
...
+
+        python3 process.py --dataset=INTL --mode=process
+        python3 process.py --dataset=ELEC --mode=process
+        python3 process.py --dataset=PET --mode=process
+        python3 process.py --dataset=NG --mode=process
+        python3 process.py --dataset=SEDS --mode=process
+        python3 process.py --dataset=NUC_STATUS --mode=process
+        python3 process.py --dataset=TOTAL --mode=process
+        
+
```
Comment (krishnaswamypradeep) [inline scripts/us_eia/opendata/README.md:60]: Please add a section to explain the behavior of the script when no mode is specified. This should include details on the default actions, which may involve both downloading and processing the data.

File: scripts/us_eia/opendata/process/coal.py [hunk 8]
```diff
@@ -84,6 +84,7 @@ def extract_place_statvar(series_id, counters):
         measure = m.group(1)
         if measure.startswith("SHIP"):
             # TODO: model destination / source port as well
+            print(series_id)
```
Comment (krishnaswamypradeep) [inline scripts/us_eia/opendata/process/coal.py:87]: Please remove this print statement if it's only used for debugging. Alternatively, use the logging module to log this.

File: scripts/us_eia/opendata/process/common.py [hunk 9]
```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Process EIA datasets to produce TMCF and CSV."""
-
+import os
+import sys
 import csv
 import json
 import logging
```
Comment (krishnaswamypradeep) [inline scripts/us_eia/opendata/process/common.py:19]: use absl module

File: scripts/us_eia/opendata/process/common.py (hunk 1 above)
Comment (krishnaswamypradeep) [reply scripts/us_eia/opendata/process/common.py:233]: Still commented lines are there

File: scripts/us_eia/opendata/process/coal.py (hunk 8 above)
Comment (Harsha-chandaluri) [reply scripts/us_eia/opendata/process/coal.py:87]: we are not doing coal as now it's on hold I will remove this file

File: scripts/us_eia/opendata/process/common.py (hunk 9 above)
Comment (Harsha-chandaluri) [reply scripts/us_eia/opendata/process/common.py]: modified

**Comment by Harsha-chandaluri** [comment]
created new PR as this PR is getting failed because of cla issue
New PR :https://github.com/datacommonsorg/data/pull/1134

**Pull request closed without merging**